# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome24/gtkextra/gtkpsfont-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- gtkextra-2-20031116/gtkextra/gtkpsfont.c.orig	2003-11-27 06:25:54.000000000 +0100
+++ gtkextra-2-20031116/gtkextra/gtkpsfont.c	2003-11-27 06:26:36.000000000 +0100
@@ -357,7 +357,7 @@
 {
   GList *list;
 
-  if(psfont_refcont <= 0) return;
+  if(psfont_refcount <= 0) return;
 
   psfont_refcount--;
 
